ipaplatform/base/services.py

```diff
@@ -483,10 +483,7 @@ def remove(self):
 
 # Objects below are expected to be exported by platform module
 
-def base_service_class_factory(name):
```

Comment:
This was removed as part of revert of your commit 8f98fa1 requested few lines above. So you need to make up your mind :-) I'm going to proceed as this is really nitpicking and our time is limited.